lib/bugsnag/integrations/rack.rb

```diff
@@ -45,6 +51,12 @@ def call(env)
       if env["rack.exception"]
         Bugsnag.notify(env["rack.exception"], true) do |report|
           report.severity = "error"
+          report.set_handled_state({
+            :type => "middleware_handler",
+            :attributes => {
+              :name => "rack"
```

Comment:
`Rack` above is capitalized, while this is not. Integration names are another candidate for a constant enumeration.